### ISSUE TITLE
user - Eternal update requirement - shift_color

### ISSUE THREAD

**Issue by SpawW** [iLert/terraform-provider-ilert#83]
Hi,
   I create users using terraform and mantain these users with same way.
   Before I create each one with a attribute "shift_color", now I try to remove this attribute... but... no luck... now every deploy... we found a requirement of update like this:
```
Terraform used the selected providers to generate the following execution
1plan. Resource actions are indicated with the following symbols:
2  ~ update in-place
3Terraform will perform the following actions:
4  # ilert_user.all_users["user@domain"] will be updated in-place
5  ~ resource "ilert_user" "all_users" {
6        id          = "xxx"
7      - shift_color = "#ff0000" -> null
8        # (8 unchanged attributes hidden)
9    }
10  # ilert_user.all_users["user@domain"] will be updated in-place
11  ~ resource "ilert_user" "all_users" {
12        id          = "xxx"
13      - shift_color = "#ff0000" -> null
14        # (8 unchanged attributes hidden)
15    }
16  # ilert_user.all_users["user@domain"] will be updated in-place
17  ~ resource "ilert_user" "all_users" {
18        id          = "xxx"
19      - shift_color = "#ff0000" -> null
20        # (8 unchanged attributes hidden)
21    }

```

My .TF:
resource "ilert_user" "all_users" {
  for_each = { for user in local.all_users : user.email => user }

  first_name = each.value.first_name
  last_name  = each.value.last_name
  email      = each.value.email
  language   = "en"
  timezone   = "<CITY/Place>"
  role       = "RESPONDER"
  position   = ""
  department = each.value.department
  # shift_color = null
}

I already test "shift_color = null"... remove tag... but... same result every time.

**Comment by STLVRTX** [comment]
Hello, thank you for submitting this issue.
It seems as if the internal state isn't properly set when applying `shift_color` to a user.
I will reproduce and confirm this.

**Comment by krystianity** [comment]
Hi @SpawW this should have been fixed in the mean time through a server side release.

**Comment by krystianity** [comment]
Closing for now, feel free to reopen

**Comment by SpawW** [comment]
@krystianity is expected to be right for now? I need to update terraform version or module? 

![image](https://github.com/iLert/terraform-provider-ilert/assets/3518964/f6bcacb0-0292-454b-86cb-e555d534bea8)

**Comment by SpawW** [comment]
Hi guys, some news? Here we still with same issue. I need to do someting to end this ?

**Comment by krystianity** [comment]
Reopening this for additional investigation, this should have been fixed by the server side release - but we will double check the provider.

**Comment by krystianity** [comment]
Closing as fixed in backend release v2.40.0
